ga4gh/server/backend.py

```diff
@@ -1726,8 +1795,8 @@ def runSearchQuery(self, request, return_mimetype, access_map):
         """
         try:
             request = protocol.fromJson(request, protocol.SearchQueryRequest)
-        except protocol.json_format.ParseError:
-            raise exceptions.InvalidJsonException(request)
+        except protocol.json_format.ParseError as e:
+            raise exceptions.InvalidJsonException(e.message)
```

Comment:
This (`e.message`) is deprecated as of 2.6, can we use something else?

Reply:
updated to str(e), seems to provide the same output